src/Ide/Plugin/Formatter.hs

```diff
@@ -65,7 +65,7 @@ doFormatting lf providers ideState ft uri params = do
       Just provider ->
         case uriToFilePath uri of
           Just (toNormalizedFilePath -> fp) -> do
-            (_, mb_contents) <- runAction ideState $ getFileContents fp
+            (_, mb_contents) <- runAction (fromNormalizedFilePath fp) ideState $ getFileContents fp
```

Comment:
These strings should probably be things like "Formatter" rather than a filepath.

Reply:
ok, so they are debugger info. They showed up in the error messages as being a filepath, I just grabbed the nearest one in scope. And it sort of made sense that you are trying to tune the graph wrt files.

But the penny did sort of drop when I saw the change in the runActionSync call in exe/Main.hs